framework/source/class/qx/lang/Number.js

```diff
@@ -91,9 +91,12 @@ qx.Class.define("qx.lang.Number",
         qx.core.Assert.assertNumber(y);
       }
 
-      // 1e-14 is the relative difference
+      // 2e-52 is the difference between 1 and the smallest floating point
+      // number greater than 1, like Number.EPSILON, but that is not supported
+      // by all browsers.
+      // 1e-14 is the relative difference.
       return x === y ||
-             Math.abs(x - y) < Number.EPSILON ||
+             Math.abs(x - y) < 2e-52 ||
```

Comment:
aah :) sorry ... what I meant is to use 2e-52 IF EPSILON was not defined

Reply:
Could we implement Number.EPSILON as a shim then, in a new `qx.lang.normalize.Number`?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/EPSILON suggests a simple polyfill:

```javascript
if (Number.EPSILON === undefined){
    Number.EPSILON  =  Math.pow(2, -52);
```

But if Number.EPSILON is 2e-52, then why can't we just use the literal constant?

Reply:
since most browsers support it ... so there is no point in ignoring it ...